middleware.js

```diff
@@ -0,0 +1 @@
+console.log('middleware');
```

Comment:
maybe be more specific for this middleware

Reply:
Yep, will fix.